release-engineering/role-handbooks/branch-manager.md

```diff
@@ -722,53 +722,51 @@ Example PRs:
 
 ### Branch Fast Forward
 
-Prior to running `./branchff`, you will need:
+Prior to running `krel ff`, you will need:
 
-- permission to write to the `/usr/local/` directory
-- permission to run under `root` privileges
 - membership in the [Release Managers GitHub Group](https://github.com/orgs/kubernetes/teams/release-managers)
 
 Command invocation:
 
 ```shell
-./branchff release-x.y
+krel ff --branch release-x.y
 ```
 
-Where `X.Y` is the is release cycle version e.g. `1.16`
+Where `X.Y` is the is release cycle version e.g. `1.18`
```

Comment:
If we're going to change the version, we need to change it throughout the document.
See the "Conventions" section: https://github.com/kubernetes/sig-release/blob/eae5668919e7508b137b45be4fc92db855740760/release-engineering/role-handbooks/branch-manager.md#conventions

Reply:
Okay, reverted that change :+1: